test/API/tfile.c

```diff
@@ -1283,30 +1287,32 @@ test_get_obj_ids(void)
 
     /* Close the file first */
     H5Fclose(fid);
-#ifndef WRONG_DATATYPE_OBJ_COUNT
-    /* Get the number of all opened objects */
-    oid_count = H5Fget_obj_count((hid_t)H5F_OBJ_ALL, H5F_OBJ_ALL);
-    CHECK(oid_count, FAIL, "H5Fget_obj_count");
-    VERIFY(oid_count, NDSETS, "H5Fget_obj_count");
+    //#ifndef WRONG_DATATYPE_OBJ_COUNT
+    if (vol_cap_flags_g & H5VL_CAP_FLAG_STORED_DATATYPES) {
```

Comment:
In this case we should probably check for H5VL_CAP_FLAG_FILE_MORE since H5Fget_obj_count and H5Fget_obj_ids are related to the file object rather than datatypes.

Reply:
Changed.